dev-docs/bidders/nativo.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: bidder
+title: Nativo
+description: Prebid Nativo Bidder Adapter
+pbjs: false
```

Comment:
@jsfledd This is for a PBJS bid adapter, so wondering why `pbjs` here is set to false?

Reply:
Thank you @jeanstemp. I mis-read the description of that parameter as "Add pbjs: true**,** If you also have a Prebid Server bid adapter." I changed the value to **true**.